source/code/source/playfab/PlayFabAndroidHttpPlugin.cpp

```diff
@@ -424,7 +424,7 @@ namespace PlayFab
                 return false;
             }
 
-            jstring urlJstr = jniEnv->NewStringUTF("POST");
+            auto urlJstr = jniEnv->NewStringUTF("POST");
```

Comment:
This doesn't meet the requirements for auto use. I don't know the return type of NewStringUTF to be a jstring.

Reply:
I know for certain that declaring the type to be jstring will compile, but will also instantly cause a seg-fault at that line of code, when executing.
I don't know what's really going on there, but auto fixes it.
I'm all for standard guidelines improving code readability, but not for when those guidelines are actively counter productive.
https://docs.oracle.com/javase/8/docs/technotes/guides/jni/spec/functions.html#NewStringUTF
For whatever reason, it "should" be jstring, but jstring causes a segfault.
Worthy of filing a separate bug, but this demonstrably fixes the segfault.

Reply:
https://dev.azure.com/playfab/PlayFab/_workitems/edit/38975